Posh-ACME/Public/Get-PAAuthorizations.ps1

```diff
@@ -71,6 +68,14 @@ function Get-PAAuthorizations {
                 Write-Warning "ACME server returned non-compliant challenge objects with no status. Please report this to your ACME server vendor."
             }
 
+            # According to RFC 8555 7.1.4 the expires property is only REQUIRED when the property status is "valid".
+            # It's OPTIONAL for any other status and some CA's will not return it.
+            # Only repair the timestamp if it actually exists
+            if([bool]($auth.PSobject.Properties.name -match "expires")) {

```

Comment:
Rather than using Regex via the `-match` operator, I'd prefer to check the existence of the property similar to how it checks for status in the challenges just above this. So:

```powershell
if ('expires' -in $auth.PSObject.Properties.Name) {
```

Reply:
I agree, thats a cleaner and better way of doing it

Reply:
In case you didn't already know, you should be able to commit the change to your local copy and push the change up to your fork on Github and it will show up here without needing to submit a new pull request.

Reply:
Thanks, its commited now